DiffusionPreprocessing/scripts/DiffusionToStructural.sh

```diff
@@ -97,28 +95,57 @@ done
 ${GlobalScripts}/Rotate_bvecs.sh "$DataDirectory"/bvecs "$WorkingDirectory"/diff2str.mat "$T1wOutputDirectory"/bvecs
 cp "$DataDirectory"/bvals "$T1wOutputDirectory"/bvals
 
+DILATE_DISTANCE=`echo "$DiffRes * 4" | bc`  # Extrapolates the diffusion data up to 4 voxels outside of the FOV
```

Comment:
Can we call this `DilateDistance` (lower case) so as to not imply that it is some sort of environment variable?

Reply:
Done